### DISCUSSION TITLE
Does JobRunner supports scheduling of jobs based on daylight timezone setting ?

### DISCUSSION THREAD

**Discussion by bhushanibm** [jobrunr/jobrunr#530] [Q&A] [answered]
There are timezone which have daylight system where in the time of triggering will change based on season. Suppose, I create a Job which needs to be scheduled to run at 2 AM every day when daylight is no on in eastern timezone. During daylight season will it trigger at 2 AM or will it schedule 1 hour prior to it ?

**Comment by bhushanibm** [comment]
@rdehuyss :- Can you please let me know your response on the above ?

**Comment by rdehuyss** [comment, marked as answer]
I didn't know it by heart but as I look into the code, I think you have two options:
- schedule it with zone is UTC: on a changé of daylight saving, it will shift from 2:00 to 1:00.
- schedule it with the time zone: on a changé of daylight saving, it will stay constant at 2:00

But again, it's a long time ago that I've written this code so best test this quickly by changing your time settings of your computer.

**Reply by rdehuyss** [reply]
Feel free to star the repo if you like JobRunr!
